src/test/java/pl/edu/pw/mini/ingreedio/api/service/ProductServiceTest.java

```diff
@@ -140,14 +157,21 @@ public void givenIngredientsCriteria_whenFilter_thenReturnCorrectProducts() {
             .ingredients(Arrays.asList("ziemniak", "burak")).build());
 
         // When
-        List<ProductDto> ziemniakBurakProducts = productService.getProductsMatching(
+        Page<Product> ziemniakBurakPage = productService.getProductsMatching(
```

Comment:
Throughout the project we use English names, let's not make exceptions for tests and stick to this convention